src/pages/KadraVykhovnykiv/KVTable.tsx

```diff
@@ -113,11 +108,11 @@ export const KVTable = ({ current, searchData }: props) => {
 
 
   return (
-    <div>
+    <div style={{textAlign:'center'}}>
```

Comment:
Is it possible to do not use inline styles?

Reply:
fixed